openassessment/templates/openassessmentblock/oa_base.html

```diff
@@ -21,12 +21,14 @@ <h2 class="openassessment__title problem__header">{% trans title %}</h2>
             <ol class="openassessment__steps">
                 {% for assessment in rubric_assessments %}
                     <li class="{{ assessment.class_id }} openassessment__steps__step is--loading">
-                        <header class="step__header">
+                        <header class="step__header ui-slidable__container">
                             <h2 class="step__title">
-                                <span class="step__counter"></span>
-                                <span class="wrapper--copy">
-                                    <span class="step__label">{% trans assessment.title %}</span>
-                                </span>
+                                <button class="ui-slidable" aria-expanded="false" disabled>
```

Comment:
This is a "dummy step" shown while the actual steps (self assessment, peer, final grade, etc.) are loading. When they finish loading, each step replaces its dummy section with its actual content. The actual content has a button for the step name, and it expands/collapses to show the content.

When I removed the color-contrast ignore, I was getting failures for steps when they are still showing this "dummy" content. The name of the step was being displayed in a pale font ("disabled looking") that doesn't meet a11y guidelines. In order to work around this and make the dummy content more consistent with the content that will replace it, I am making this a button and setting it to disabled. The font will be the regular dark gray (accessible) font, but because the button is disabled, the browser changes it to a reasonable disabled color. Before and after pictures are below (before had lighter step label text).
<img width="860" alt="before" src="https://cloud.githubusercontent.com/assets/484484/20153916/054ef01c-a693-11e6-986b-1852742925af.png">

<img width="846" alt="after" src="https://cloud.githubusercontent.com/assets/484484/20153922/0bd05fa2-a693-11e6-9597-b66c8be911b8.png">